basculehelper/basculeHelper.go

```diff
@@ -0,0 +1,510 @@
+package basculehelper
+
+//Deprecated: this is a bascule helper package that uses older bascule functions from webpa-common in order to implement
+//sallust and zap logger in scytale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"emperror.dev/emperror"
+	"github.com/SermoDigital/jose/jwt"
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/provider"
+	"github.com/spf13/cast"
+	"github.com/xmidt-org/bascule"
+	"github.com/xmidt-org/bascule/basculehttp"
+	"github.com/xmidt-org/webpa-common/v2/xmetrics"
+	"go.uber.org/fx"
+)
+
+var (
+	ErrNoVals                 = errors.New("expected at least one value")
+	ErrNoAuth                 = errors.New("couldn't get request info: authorization not found")
+	ErrNoToken                = errors.New("no token found in Auth")
+	ErrNoValidCapabilityFound = errors.New("no valid capability for endpoint")
+	ErrNilAttributes          = errors.New("nil attributes interface")
+	ErrNoURL                  = errors.New("invalid URL found in Auth")
+	partnerKeys               = []string{"allowedResources", "allowedPartners"}
+)
+
+const (
+	CapabilityKey = "capabilities"
+
+	RejectedOutcome = "rejected"
+	AcceptedOutcome = "accepted"
+	// reasons
+	TokenMissing               = "auth_missing"
+	UndeterminedPartnerID      = "undetermined_partner_ID"
+	NoCapabilityChecker        = "no_capability_checker"
+	EmptyParsedURL             = "empty_parsed_URL"
+	AuthCapabilityCheckOutcome = "auth_capability_check"
+	AuthValidationOutcome      = "auth_validation"
+	capabilityCheckHelpMsg     = "Counter for the capability checker, providing outcome information by client, partner, and endpoint"
+	EmptyCapabilitiesList      = "empty_capabilities_list"
+	MissingValues              = "auth_is_missing_values"
+	UndeterminedCapabilities   = "undetermined_capabilities"
+	NoCapabilitiesMatch        = "no_capabilities_match"
+	TokenMissingValues         = "auth_is_missing_values"
+
+	//labels
+	OutcomeLabel   = "outcome"
+	ReasonLabel    = "reason"
+	ClientIDLabel  = "clientid"
+	EndpointLabel  = "endpoint"
+	PartnerIDLabel = "partnerid"
+	ServerLabel    = "server"
+)
+
+/****************************AuthCapabilitiyCheckMeasures****************************/
+
+// AuthCapabilityCheckMeasures describes the defined metrics that will be used by clients
+type AuthCapabilityCheckMeasures struct {
+	CapabilityCheckOutcome metrics.Counter
+}
+
+// NewAuthCapabilityCheckMeasures realizes desired metrics. It's intended to be used alongside Metrics() for
+// our older non uber/fx applications.
+func NewAuthCapabilityCheckMeasures(p provider.Provider) *AuthCapabilityCheckMeasures {
+	return &AuthCapabilityCheckMeasures{
+		CapabilityCheckOutcome: p.NewCounter(AuthCapabilityCheckOutcome),
+	}
+}
+
+/****************************AuthValidationMeasures****************************/
```

Comment:
Comments like this aren't idiomatic go.  We should avoid them, and instead make sure that comments for types adhere to the `golang` standard: https://go.dev/doc/comment

I think all the comments on types in this package already adhere to that standard, so we can just delete these /* ... */ comments.